app/views/products/_form.html.erb

```diff
@@ -31,6 +31,21 @@
     <%= form.text_field :price %>
   </div>
 
+  <div>
+    <%= form.label :discount_price, style: "display: block" %>
```

Comment:
Why have you used `style: "display: block"` here?

Reply:
It was given like that in the book, there was inline styling in other fields too
Removed inline styling